src/clarity-angular/wizard/wizard-button.ts

```diff
@@ -107,7 +107,7 @@ export class WizardButton {
         let nav = this.navService;
```

Comment:
i know this isn't touched in this PR but it looks like `nav` isn't being used anywhere, no?